docs/cloud-config/index.md

```diff
@@ -87,3 +87,12 @@ rancher:
         - 8.8.4.4
 
 ```
+
+#### Set Hostname
+
+You can set the hostname of the host using cloud-config. The example below shows how to configure it.
+
+```yaml
+hostname: myhost
+```
```

Comment:
Never mind on my comment based on our discussion that this is how to overwrite the "rancher" hostname for RancherOS.